elements/pfe-clipboard/src/pfe-clipboard.js

```diff
@@ -39,6 +39,12 @@ class PfeClipboard extends PFElement {
         attr: "no-icon",
         observer: "_noIconChanged"
       },
+      copiedDuration: {
+        title: "Copied Duration",
+        type: Number,
+        attr: "copied-duration",
```

Comment:
This is the default for the attribute name so you won't need to define it here.  Since you used camel case, the properties parsing will automatically assumed `copied-duration` is the name of the attribute.

Reply:
Noice! Removed.  I'll remove the one on no-icon as well.